ml/db/init.sql

```diff
@@ -0,0 +1,170 @@
+-- INITALIZATION
+
+DROP DATABASE stratosphere;
+
+CREATE DATABASE stratosphere;
+ALTER DATABASE stratosphere OWNER TO postgres;
+
+\connect stratosphere
+
+ALTER USER postgres PASSWORD 'test1';
```

Comment:
In the future, don't include the passwords. I know this is part of my docker ymls, what will be happening is that these will go into a secret file and be referred there.